skipper.go

```diff
@@ -978,6 +978,7 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 			loadbalancer.HealthcheckPostProcessor{LB: lbInstance},
 			loadbalancer.NewAlgorithmProvider(),
 			scheduler.NewRegistry(),
+			routing.NewFilterCreationMetrics(mtr),
```

Comment:
Considering that this change has a potentially significant change to the memory usage, please make it an opt-in feature for both the library and the command line distributions by creating an option and a flag for it.